include/llvm/Config/config.h

```diff
@@ -17,7 +17,9 @@
 #define LLVM_ENABLE_CRASH_DUMPS 0
 
 /* Define to 1 if you have the `backtrace' function. */
+#if !defined(__ANDROID__)
 #define HAVE_BACKTRACE TRUE
+#endif
```

Comment:
Pretty much the same as [an earlier patch for llbuild](https://github.com/apple/swift-llbuild/pull/553/files#diff-2675b3ae9211823220c66e10862f930c).